<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDriverTables</class>
 <widget class="QWidget" name="FormDriverTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Контекст драйвера</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tvDriverContext"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pbDrContextSubmitAll">
         <property name="text">
          <string>Записать изменения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbDrContextRevertAll">
         <property name="text">
          <string>Отменить изменения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Контекст тэга</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tvTagContext"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="pbTagContextSubmitAll">
         <property name="text">
          <string>Записать изменения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbTagContextRevertAll">
         <property name="text">
          <string>Отменить изменения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
